crescent/bot.py

```diff
@@ -108,6 +108,8 @@ def __init__(
         self.command_hooks = command_hooks
         self.command_after_hooks = command_after_hooks
 
+        self._started = False
+
```

Comment:
Might be worth using an asyncio.Future instead of a bool. This would also allow you to `await bot._started` to wait until the bot is started.